src/test/java/io/seqera/tower/cli/BaseCmdTest.java

```diff
@@ -48,7 +48,24 @@
 
     protected byte[] loadResource(String name) {
         try {
-            return this.getClass().getResourceAsStream("/runcmd/" + name + ".json").readAllBytes();
+            InputStream stream = this.getClass().getResourceAsStream("/runcmd/" + name + ".json");
```

Comment:
this method should call `loadResource(name, "json")` to prevent code duplication